source/agora/consensus/Quorum.d

```diff
@@ -66,32 +66,33 @@ public struct QuorumParams
     to the quorum assignment of each node.
 
     Params:
-        key = the key of the node for which to generate the quorum
+        key = the `PublicKey` of the node
+        utxo_key = the key of the node for which to generate the quorum
         utxo_keys = the list of UTXO keys of all the active enrollments
         finder = UTXO finder delegate
         rand_seed = the random seed
         params = quorum generator algorithm tweaking parameters
 
 *******************************************************************************/
 
-public QuorumConfig buildQuorumConfig (in PublicKey key,
+public QuorumConfig buildQuorumConfig (in PublicKey key, in Hash utxo_key,
```

Comment:
Why do we still need the public key here?

Reply:
I think I should remove the public key. Will update.

Reply:
Updated